src/runtime_src/core/edge/drm/zocl/zocl_drv.c

```diff
@@ -910,6 +905,17 @@ static int zocl_drm_platform_probe(struct platform_device *pdev)
 
 	platform_set_drvdata(pdev, zdev);
 
+        int year, mon, day;
```

Comment:
You may want to put this line at the beginning of the function. Otherwise you will see warning: "ISO C90 forbids mixed declarations and code......"
Also, it seems having ident issue.

Reply:
Done